components/KurdCard.tsx

```diff
@@ -15,31 +16,32 @@ const { Meta } = Card;
 
 export default function KurdCard({ kurd }: { kurd: KurdWithTopics }) {
   return (
-    <a href={kurd.link} target="_blank" rel="noreferrer">
+    <a href={kurd.link} target='_blank' rel='noreferrer'>
       <Card
         className={styles.card}
         cover={
           <Image
             className={styles.avatar}
             src={getPhoto(kurd)}
             alt={kurd.name}
-            preview={false}
+            height={250}
+            width={250}
           />
         }
         actions={[
-          <a key="social" target="_blank" href={kurd.link} rel="noreferrer">
+          <a key='social' target='_blank' href={kurd.link} rel='noreferrer'>
```

Comment:
Actually, why are the double quotes gone?

Reply:
Well, Thanks to my Prettier extension, it refactors my code on saving, I prefer single quotes over double! 😄

Reply:
Cool. So this repo needs to configure your Prettier settings, as well. :)

Reply:
sorry, my bad, I didnt notice that until I did the pull request.

Reply:
Not a problem, though. You can still push more commits.

Reply:
maybe we can now have a code refactoring policy and add a Prettier config file in order to prevent this from happening again.

Reply:
Yes! I said that [here](https://github.com/hezhamen/awesome-kurds/pull/23#discussion_r775845042).

Reply:
I thought you're joking 😆

Reply:
@roj1512  I reverted the single quotes back to the double quotes in my last commit.

Reply:
Super!